config.yaml

```diff
@@ -562,6 +562,29 @@ usage_collector:
     # Sets the interval between running the usage collector in days
     interval_in_days: 1
 
+prometheus:
+  port: 9090
+  #services_to_monitor:
+  #  - node
+  #  - blackbox
+  #  - postgresql
+  #  - rabbitmq
```

Comment:
@mateumann Do not we need to add them to the config ? why they are commented out ?

Reply:
no particular reason (for the code currently dealing with Prometheus+exporters intstalation);  I had some weird idea about cluster deployment, but that could wait till the next PR